packages/iTwinUI-react/src/core/Table/Table.tsx

```diff
@@ -694,10 +694,7 @@ export const Table = <
                       { 'iui-sorted': column.isSorted },
                       column.columnClassName,
                     ),
-                    style: {
-                      ...getCellStyle(column, !!state.isTableResizing),
-                      flexWrap: 'unset',
-                    },
+                    style: getCellStyle(column, !!state.isTableResizing),
```

Comment:
In CSS we are adding a new element `<div className='iui-table-header-wrapper'>`. Don't we need that here?